packages/client/components/ScopePhaseArea.tsx

```diff
@@ -117,6 +115,38 @@ const ScopePhaseArea = (props: Props) => {
     setActiveIdx(2)
   }
 
+  const contents: Record<typeof baseTabs[number]['label'], JSX.Element> = {
```

Comment:
+2 i like the refactor!
+1 you could take this one step further by putting the component in the `tabs` array (just make Jira accept a `meetingRef` and ignore passing `gotoParabol` into the parabol tab)

```ts
 const baseTabs = [
    {icon: <GitHubSVG />, label: 'GitHub', allow: true, Component: ScopePhaseAreaGitHub}
]

tabs.map(({label, Component}) => <Component meeting={meeting} isActive={isTabActive(label)}, gotoParabol={gotoParabol}/>
```